src/page-modules/assistant/layout.tsx

```diff
@@ -82,6 +86,11 @@ function AssistantLayout({ children, tripQuery }: AssistantLayoutProps) {
     setIsSwapping(false);
   };
 
+  const onClear = () => {
```

Comment:
Naming this something like `onClearViaLocation` would make this more readable I think.

Reply:
Agree🤔